lib/buildpack/compile/installers/installer.rb

```diff
@@ -56,6 +56,10 @@ def should_install(_app_dir)
       false
     end
 
+    def version_file
```

Comment:
This method was originally intended to only be used internally by some of the installers (the Node installer does not use this), is there a reason to make it public rather than protected?